packages/appstore/src/assets/svgs/banner/svg-migrate-mobile.svg

```diff
@@ -0,0 +1 @@
7.352.012.7-.104 1.044a2.892 2.892 0 0 1-.611 1.004c-.286.322-.667.621-1.145.897a5.18 5.18 0 0 1-.617.3 6.383 6.383 0 0 1-.646.249 7.308 7.308 0 0 1-.575.18 2.325 2.325 0 0 1-.426.079l-.457-1.44c.217-.024.49-.08.818-.168a5.025 5.025 0 0 0 1.078-.484c.478-.276.771-.57.877-.88.103-.318.081-.606-.067-.862ZM337.288 47.326l-2.079-3.601 2.411-1.392.393.68-1.601.924.435.754 1.424-.822.393.68-1.424.823.858 1.486-.81.468ZM338.192 42.002l.811-.468 2.079 3.601-.811.468-2.079-3.6ZM344.332 43.259c-.471-.278-.957-.54-1.459-.786a14.364 14.364 0 0 0-1.519-.64l1.362 2.359-.801.462-2.079-3.601.66-.381c.181.048.388.116.623.202.234.086.477.183.729.292.254.103.511.216.771.338.258.119.503.236.736.351l-1.227-2.125.805-.465 2.079 3.6-.68.394Z"/></g><path fill="url(#t)" d="M0 0h170v120H0z" transform="rotate(180 165 60)"/></g><defs><clipPath id="a"><path fill="#fff" d="M166 1h154a7 7 0 0 1 7 7v92a7 7 0 0 1-7 7H166V1Z"/></clipPath><clipPath id="b"><path fill="#fff" d="m126.801 38.626 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="c"><path fill="#fff" d="m147.602 74.653 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="d"><path fill="#fff" d="m168.398 110.68 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="e"><path fill="#fff" d="m142.027-18.2 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="f"><path fill="#fff" d="m162.828 17.826 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="g"><path fill="#fff" d="m183.629 53.853 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="h"><path fill="#fff" d="m204.426 89.88 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="i"><path fill="#fff" d="m198.855-2.974 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="j"><path fill="#fff" d="m219.656 33.054 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="k"><path fill="#fff" d="m240.453 69.08 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="l"><path fill="#fff" d="m261.254 105.106 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="m"><path fill="#fff" d="m234.879-23.774 27.713-16 16 27.712-27.713 16z"/></clipPath><clipPath id="n"><path fill="#fff" d="m255.68 12.253 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="o"><path fill="#fff" d="m276.477 48.28 27.713-16 16 27.712-27.713 16z"/></clipPath><clipPath id="p"><path fill="#fff" d="m297.277 84.306 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="q"><path fill="#fff" d="m318.078 120.332 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="r"><path fill="#fff" d="m291.707-8.547 27.713-16 16 27.713-27.713 16z"/></clipPath><clipPath id="s"><path fill="#fff" d="m312.504 27.48 27.713-16 16 27.712-27.713 16z"/></clipPath><linearGradient id="t" x1="165" x2="5.506" y1="106" y2=".718" gradientUnits="userSpaceOnUse"><stop offset=".232" stop-color="var(--general-main-1)"/><stop offset=".646" stop-color="var(--general-main-1)" stop-opacity=".651"/><stop offset="1" stop-color="var(--general-main-1)" stop-opacity="0"/></linearGradient></defs></svg>
```

Comment:
Did we run svgo? the icons don't seem to be minified!

Reply:
yes, both mobile and desktop images were `svgo` optimized before pushing to git. 🗄️

Reply:
it was 110 kb. now its 57 kb.